lib/broccoli/amd-shim.js

```diff
@@ -6,11 +6,14 @@ module.exports = function shimAmd(tree, nameMapping) {
     let name = nameMapping[relativePath];
     if (name) {
       return [
-        '(function(define){\n',
+        '(function(define, module){\n',
```

Comment:
In rewriting this for the last comment, I am slightly confused about this part:
```
while (typeof args[0] === "string") { args.shift(); }
```

What is this trying to do?  Why would `args[0]` (or an arbitrary number of initial arguments) be strings?